controllers/admin_domains.php

```diff
@@ -505,6 +507,17 @@ public function importPackages()
                 }
             }
 
+            // Get generic domain module
```

Comment:
This should not be.  We want to use the same module as on the old package.  To that end we could update $created_tlds to be a mapping of [tld => [package_id, module_id]] where module_id = $package->module_id